IPlug/IGraphics/IGraphicsStructs.h

```diff
@@ -27,14 +27,29 @@ enum EDirection { kVertical, kHorizontal };
  * An IBitmap's width and height are always in relation to a 1:1 (low dpi) screen. Any scaling happens at the drawing stage. */
 struct IBitmap
 {
+  /** Pointer to the raw bitmap data */
   void* mData;
+  /** Bitmap width (in pixels) */
   int W;
+  /** Bitmap height (in pixels) */
   int H;
-  /** Number of states (for multibitmaps) */
+  /** Number of frames (for multibitmaps) */
   int N;
+  /** \c True if the frames are positioned horizontally */
   bool mFramesAreHorizontal;
-  double mSourceScale; // i.e. highest res available for this resource
+  /** Maximum scaling allowed for the bitmap (typically 1) */
```

Comment:
Please check if this is correct before merging

Reply:
Bitmap stuff is being worked on currently, will make sure docs are correct when it is finialised! thanks